uberduck_ml_dev/exec/train_tacotron2.py

```diff
@@ -1,16 +1,36 @@
 # AUTOGENERATED! DO NOT EDIT! File to edit: nbs/exec.train_tacotron2.ipynb (unless otherwise specified).
 
-__all__ = ['run']
+__all__ = ['parse_args', 'run']
 
 # Cell
 from ..trainer.tacotron2 import Tacotron2Trainer
 from ..vendor.tfcompat.hparam import HParams
 from ..models.tacotron2 import DEFAULTS as TACOTRON2_DEFAULTS
-from ..utils.argparse import parse_args
+import argparse
 import sys
 import json
 
 # Cell
+def parse_args(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config", help="Path to JSON config")
+    parser.add_argument(
+        "--training_audiopaths_and_text", help="Path to training filelist", default=None
+    )
+    parser.add_argument(
+        "--val_audiopaths_and_text", help="Path to val filelist", default=None
+    )
+    parser.add_argument("--log_dir", help="Path to log_dir", default=None)
+    parser.add_argument(
+        "--checkpoint_path", help="Path to checkpoint_path", default=None
+    )
+    parser.add_argument(
+        "--warm_start_name", help="Path to warm start model", default=None
+    )
+    args = parser.parse_args(args)
+    return args
```

Comment:
I think this is a reasonable approach for now. Down (aka once we have validation for the config JSON schema) the line I think it'd make sense to do something like https://stackoverflow.com/a/37367814 so that we can allow anything in the config to be overridden without explicitly specifying it.